lib/ReactViews/Mobile/MobileMenu.jsx

```diff
@@ -77,12 +79,41 @@ const MobileMenu = observer(
       this.props.viewState.toggleFeaturePrompt("mapGuidesLocation", true, true);
     },
 
+    /**
+     * If the help configuration defines an item named `mapuserguide`, this
+     * method returns props for showing it in the mobile menu.
+     */
+    mapUserGuide() {
```

Comment:
:/ this seems really hacky, what if we instead moved this into the map specific code ie similar to how the About link is generated ie inside UserInterface.jsx:35 inside TerriaMap? That way, the responsibility of the translation is also deferred to the map specific translation code and we don't have any special treatment for the link inside terrirajs

Reply:
Also the external icon link should be right aligned in the mobile menu as per Phil's designs

Reply:
Yes it is a bit hacky, but I think we should treat this as a stop gap before doing https://github.com/TerriaJS/terriajs/issues/5845 as part of the larger mobile UI rework.

Good point about the icon alignment. I'll fix it.